openmetadata-ui/src/main/resources/ui/src/pages/TableDetailsPageV1/TableDetailsPageV1.tsx

```diff
@@ -733,15 +733,14 @@ const TableDetailsPageV1: React.FC = () => {
           <TabsLabel id={EntityTabs.DBT} name={t('label.dbt-lowercase')} />
         ),
         isHidden: !(
-          tableDetails?.dataModel?.sql ?? tableDetails?.dataModel?.rawSql
+          tableDetails?.dataModel?.sql || tableDetails?.dataModel?.rawSql
         ),
         key: EntityTabs.DBT,
         children: (
           <QueryViewer
             sqlQuery={
-              tableDetails?.dataModel?.sql ??
-              tableDetails?.dataModel?.rawSql ??
-              ''
+              get(tableDetails, 'dataModel.sql', '') ||
```

Comment:
Would recommend a single useMemo variable and use it in both places.

Reply:
` tableDetails?.dataModel?.sql || tableDetails?.dataModel?.rawSql` by using this it will return `undefined or query`
and for this `get(tableDetails, 'dataModel.sql', '') || get(tableDetails, 'dataModel.rawSql', '')` it will return empty string or query due to which we can not use single variable